tests/strategies/test_environ.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import os
+import pytest
```

Comment:
to pass on `isort` tests you need to add a line break between `os` and `pytest` because `os` is stdlib and `pytest` is 3rd party

Reply:
@drgarcia1986 I added a line break, tests still failed. What else could be wrong? Thanks.